pulp_smash/tests/pulp3/pulpcore/api_v3/test_crud_repos.py

```diff
@@ -43,7 +43,7 @@ def test_02_read_repo(self):
     def test_02_read_repos(self):
         """Search for the repository by its name.
 
-        Assert that just one search result is returned, and that that result
+        Assert that just one search result is returned, and that result
```

Comment:
This is a nice grammar improvement, but it's not related to the rest of this test. While we're at it, it'd be good to change the line wrapping:

    Assert that just one search result is returned, and that result has a
    correct name.

Reply:
@Ichimonji10 , done. If there is any minor update in the future will be in a separate PR.